packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,111 @@
+2. Apply the `.pf-theme-dark` class to the browser's `prefers-color-scheme` media query. 
+
+### React 18
+We now support the latest version of React, which was released last year. 
+
+### Versioned CSS variables and class names
+We've updated our CSS variables and class names so that they are explicitly associated with the version of PatternFly that they align to. 
+
+For more details, refer to [our upgrade guide](/get-started/upgrade#review-and-update-variable-and-class-names) and our [updated CSS classes and variables](developer-resources/global-css-variables) documentation.
+
+## Most impactful changes
+This section highlights the most impactful changes that were made to our React and HTML libraries. For a full, detailed list of changes, refer to [our upgrade guide.](/get-started/upgrade)
+
+### Event handler 
+
+Per customer request, and to meet industry standards, we’ve updated all callback signatures for component event handling properties so that the event is the first parameter. This change will make it easier to incorporate third party libraries, especially in the case of form building tools such as Formik.
+
+Our [Codemods](/get-started/upgrade#run-our-codemods) should take care of most of these changes, but if your callback function is defined at the class level you will neeed to manually update your code.
+
+### Icons
+
+We are no longer using inline styles, color, or size for icons. Now, React icons come with a built in class `pf-v5-svg`, which gives an icon a default height, width, and color.
+
+If you want a custom color or size, wrap your icon in the `Icon` React component, which comes with colors, size, etc.
+
+**Link to updated readme with instructions on how to override CSS if not using React Icon component.
+**
+
+### Deprecated components 
+We've deprecated some of our React components, which required heavy and rigid implementation processes. Our recommendations and replacements offer more versatile methods for implementing and utilizing each component. 
+
+The following table outlines the components that we deprecated, as well as the component(s) that we recommend using as a replacement.  
+
+| Component  | Recommended replacement  |
+| --- | --- |
+| Application launcher  | Dropdown, select (new) |
+| Context selector  | Menu   |
+|  Dropdown |  Dropdown (new)  |
+| Options menu  |  Dropdown (new), select (new) |
+| `PageHeader`, `PageHeaderTools`, `PageHeaderToolsGroup`, `PageHeaderToolsItem`,  | Masthead   |
+| Select  | Select (new)   |
+|  Table | Table (new) |
+| Wizard | Wizard (new) |
+
+### **Helper text**
+
+Previously, a few components utilized `helperText` properties to provide users with context regarding field inputs. Now, you will need to use the helper text component to provide this context for the following components:
+
+- [Form](/get-started/upgrade#form)
+- [Date picker](/get-started/upgrade#date-picker)
+- [Time picker](/get-started/upgrade#time-picker)
+
+### **Components**
+
+#### **Card**
```

Comment:
Shouldn't this just be part of the highlights section? It would make sense to also talk about design changes for cards and maybe include some visuals.